build.sh

```diff
@@ -15,10 +15,10 @@ esac
 done
 
 echo "Extracting configuration file..."
-if [ ! -f ~/.config/cftool ]; then
-    mkdir -p ~/.config/cftool/
+if [ ! -f $HOME"/.Appliscale/cftool" ]; then
+    mkdir -p $HOME"/.Appliscale/cftool"
 fi
-cp config.yaml ~/.config/cftool/config.yaml
+cp config.yaml $HOME"/.Appliscale/cftool/config.yaml"
```

Comment:
Okay so once again:

`~/.config/cftool/config.yaml` this path was good, but we would like to:

1. Wrap it in double quotes `"`.
2. Replace `~` with `$HOME`.

`"$HOME/.config/cftool/conifg.yaml"`.

Why wrap variables in double quotes? Problem is with `bash` globbing e.g. stars, dots etc. - without quotes it will perform it without your knowledge.

Reply:
Ok, now I got your point :)